### PR TITLE
[CHF-374] Better healthPoll scheduling for Cree Bulb

### PR DESCRIPTION
**Pull request by jackchi**
SchedulingAPIs added to DTH
https://github.com/PhysicalGraph/data-management/pull/3204

Currently it is on the data-mgmt dev environment. This is ready to be synced up with it. 

@bflorian @workingmonk
@mortent @dchem

### CODE REVIEW

File: devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy [hunk 1]
```diff
@@ -97,13 +97,14 @@ def refresh() {
...
 def healthPoll() {
+    log.debug "healthPoll()"
     def cmds = zigbee.onOffRefresh() + zigbee.levelRefresh()
     cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
 }
 
 def configure() {
     unschedule()
-    schedule("0 0/5 * * * ? *", "healthPoll")
+    runEvery5Minutes("healthPoll")
```
Comment (workingmonk) [inline devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy:107]: do we support this in device types now?

**Comment by jackchi** [comment]
Yes the PR linked added support

Jack Chi, 415-216-3240
Software Engineer

**Comment by workingmonk** [comment]
in that case it looks good. but do not merge until that change gets to production otherwise it will cause failures during cree configure

**Comment by jackchi** [comment]
This change is for dev. We want to merge it into master and then staging.

**Comment by workingmonk** [comment]
👍

**Pull request merged**